low-level/sspl_ll_d

```diff
@@ -715,6 +717,15 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
+        EventMessage.init(component='sspl', source='S')
+        logger.debug("IEM framework initiated successfully!!!")
```

Comment:
let this be logger.info(), useful info to see during sspl bootup

Reply:
done